lib/practice_exercises.rb

```diff
@@ -1,13 +1,38 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(n)
 def remove_duplicates(list)
```

Comment:
That works!  The time complexity is correct, but you are not building a new data structure so your space complexity is not correct.  Instead that is O(1).